src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  padding: 10px 0;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.link-logo {
+  margin-left: 50px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav_list {
```

Comment:
The class name `.nav_list` should be `.nav__list` to maintain consistency with BEM naming conventions.